src/Views/Admin/Teams/Addteam/AdTeam.jsx

```diff
@@ -32,36 +31,43 @@ const AdTeam = () => {
    * the upload of an image file and set the image preview accordingly.
    */
 
-  const { setImageUpload, uploadFile, isLoading, bannerURL } =
-    bannerUploadHandler();
   /** Code Above is the Handler for Banner Uploads.  It will return the Image File's downloadURL and put it into bannerURL.*/
 
-  const { LOGO_setImageUpload, LOGO_uploadFile, LOGO_isLoading, logoURL } =
-    logoUploadHandler();
   /** Code Above is the Handler for Logo Uploads. It will return the Image File's downloadURL and put it into logoURL. */
-
-  const handleImageUpload = (event) => {
+  const {
+    uploadFile2,
+    logoFile,
+    setLogoFile,
+    bannerFile,
+    setBannerFile,
+    uploading,
+    setUploading,
+  } = UploadHandler();
```

Comment:
Let's try to avoid deconstructing unneeded variables. 

We can just call UploadHandler(path, fileName);

only these parameters are need by the uploadHandlerService.

Reply:
![image](https://github.com/Hitoru141/EsportsWebsite/assets/101559620/04a82597-9ca7-447c-b1d9-ab53c8e55f0d)